tests/sentry/ingest/billing_metrics_consumer/test_billing_metrics_consumer_kafka.py

```diff
@@ -22,15 +24,16 @@ def test_outcomes_consumed(track_outcome):
 
     topic = Topic("snuba-generic-metrics")
 
-    buckets = [
+    empty_tags: dict[str, str] = {}
+    buckets: list[MetricsBucket] = [
         {  # Counter metric with wrong ID will not generate an outcome
             "metric_id": 123,
             "type": "c",
             "org_id": 1,
             "project_id": 2,
             "timestamp": 123,
             "value": 123.4,
-            "tags": {},
+            "tags": empty_tags,
```

Comment:
Not sure if the linter should fail here... 🤔

Reply:
this is a microbug with mypy -- it doesn't know that `{}` (`dict[<nothing>, <nothing>]`) solves the variance for `Mapping[K, V]`